libs/openchallenges/org-search/src/lib/org-search-filters-values.ts

```diff
@@ -0,0 +1,19 @@
+import { FilterValue } from '@sagebionetworks/openchallenges/ui';
+
+export const organizationRolesFilterValues: FilterValue[] = [
```

Comment:
```suggestion
export const contributionRolesFilterValues: FilterValue[] = [
```

The term "contribution" gives us a bit more flexibility as this opens the door to handling organizations that contributed to challenges in a non-organization way, e.g. as participant.